services/web-app/components/catalog-search/catalog-search.js

```diff
@@ -40,16 +35,7 @@ const CatalogSearch = ({ onSearch }) => {
               size="lg"
             />
           </Column>
-          <Column className={styles.column} sm={2} md={4} lg={4}>
-            <MultiSelect
-              id="catalog-filter"
-              label="Filters"
-              items={items}
-              itemToString={(item) => (item ? item.text : '')}
-              initialSelectedItems={[items[0], items[1]]}
-              size="lg"
-            />
-          </Column>
+          <CatalogMultiselectFilter onSelect={onSelect} />
```

Comment:
I'd leave the `<Column>` here. Ideally, a component simply fills its parent's container. Also, it's not like this component will be used elsewhere, but if it will be, it would be nice if it didn't have it's own layout logic. That should belong to it's parent.